dll/win32/winmm/winemm.h

```diff
@@ -205,4 +203,9 @@ extern HANDLE psStopEvent;
 
 INT LoadRegistryMMEDrivers(char* key);
 
+// REACTOS:
+#define ARRAY_SIZE(a) (sizeof(a) / sizeof((a)[0]))
```

Comment:
How about using `_countof` ?

Reply:
I copied this from wine. It's the same thing, really.

Reply:
OK.
Should I replace `_countof` in my winmm PR with `ARRAY_SIZE` after this PR merged.

Reply:
I suggest doing so, because it will probably make it easier to get changes merged upstream. Btw, did you try to sync the files first, before changing them? I only synced mci, because it was easy (the rest is more complicated) and it had all the fixes I needed at this point. But generally we should try to sync it as much as possible.

Reply:
> did you try to sync the files first, before changing them?

No, I didn't do that.
My changes are completely different from Wine because I have to do some refactoring to get the system sound to work. The reason behind the changes is to support user impersonation that Wine doesn't have to deal with.

Note: I only changed `midimap.c` and `playsound.c`.